### ISSUE TITLE
Adding `linux_aarch64`

### ISSUE THREAD

**Issue by jakirkham** [conda-forge/flash-attn-feedstock#23]
Currently this is on `linux_64` only. Though it would be nice to have this available for `linux_aarch64` too.

In the past this was blocked on `pytorch`. However `pytorch` added `linux_aarch64` in PR: https://github.com/conda-forge/pytorch-cpu-feedstock/pull/224

So perhaps this is now doable?

**Comment by weiji14** [comment]
Just checked upstream and it should be possible since https://github.com/Dao-AILab/flash-attention/pull/757 is merged (flash-attn v2.5.7 or higher). Do you want to give this a try?

**Comment by jakirkham** [comment]
Thanks Wei! 🙏

Can certainly give this a go and see how far I get

**Comment by jakirkham** [comment]
@conda-forge-admin , please re-render
